### PR TITLE
enh(refacto): refacto breadcrumb component

### PR DESCRIPTION
**Pull request by kduret**

### CODE REVIEW

File: src/Breadcrumb/index.test.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-undef */
```
Comment (bdauria) [inline src/Breadcrumb/index.test.jsx]: This should be fixed by the newly built eslint-config-centreon

File: src/Breadcrumb/Link.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import PropTypes from 'prop-types';
+import Link from '@material-ui/core/Link';
+import { Link as RouterLink } from 'react-router-dom';
+
+const SmallLink = styled(Link)(() => ({
+  fontSize: '12px',
+}));
+
+const BreadcrumbLink = ({ index, count, breadcrumb }) => {
+  return (
```
Comment (bdauria) [inline src/Breadcrumb/Link.jsx]:
```suggestion
  const isLastLink = index === count - 1;
  
  return (
```

File: src/Breadcrumb/Link.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    <SmallLink
+      color={index !== count - 1 ? 'inherit' : 'textPrimary'}
```
Comment (bdauria) [inline src/Breadcrumb/Link.jsx]:
```suggestion
      color={isLastLink ? 'textPrimary': 'inherit' }
```

File: src/Breadcrumb/index.test.jsx (hunk 1 above)
Comment (kduret) [reply src/Breadcrumb/index.test.jsx]: waiting https://github.com/centreon/eslint-config-centreon/pull/3 to be merged

**Pull request merged**